### PR TITLE
Use one output tombstone value

### PR DESCRIPTION
**Pull request by tranma**
! @jystic 
tombstone will be an ice argument, or defaults to NA

### CODE REVIEW

File: icicle-compiler/src/Icicle/Benchmark.hs [hunk 1]
```diff
@@ -103,7 +104,7 @@ createBenchmark mode dictionaryPath inputPath outputPath dropPath packedChordPat
 
   let cfg = HasInput
           ( FormatPsv (PsvInputConfig  mode input')
-                      (PsvOutputConfig mode output'))
+                      (PsvOutputConfig mode output' defaultOutputTombstone))
```
Comment (jystic) [inline icicle-compiler/src/Icicle/Benchmark.hs]: As per @raronson's comments in the ice PR, we should call this missing value

**Comment by jystic** [comment]
Looks good 👍

**Comment by tranma** [comment]
@raronson note that this applies to arrays/maps/structs too, so `feature injury ~> latest 2 ~> action` gives
`homer|test|["ignore",NA]`
rather than
 `homer|test|["ignore"]`

given
```
homer|injury|{"location":"torso","severity":3,"action":"ignore"}"|1999-01-01
homer|injury|{"location":"torso","severity":1}"|2010-01-01
```

**Comment by jystic** [comment]
> @raronson note that this applies to arrays/maps/structs too

I'm not sure if this is right, as the result won't be valid JSON anymore

**Comment by tranma** [comment]
hipspace discussion - leaving this as-is until we figure out how we want to let the user control the behaviour of tombstones (bottom/maybe/combination of both, etc.)

**Pull request merged**